user_tools/src/spark_rapids_pytools/wrappers/databricks_aws_wrapper.py

```diff
@@ -41,6 +41,9 @@ def qualification(cpu_cluster: str = None,
                           QualGpuClusterReshapeType.get_default()),
                       jvm_heap_size: int = 24,
                       verbose: bool = False,
+                      cpu_discount: int = None,
+                      gpu_discount: int = None,
+                      global_discount: int = None,
```

Comment:
Should the default values of discounts be 0?

Reply:
We can remove the checks for 'if <> is None' in that case

Reply:
I had the same idea, but in that case we cannot distinguish a user input discount = 0 from our default value. So I keep it as `None` so we would know that the option is not specified by the user.

Reply:
That makes sense. Thanks @cindyyuanjiang